src/main/java/net/minecraftforge/registries/GameData.java

```diff
@@ -482,7 +474,7 @@ public void onAdd(IForgeRegistryInternal<Item> owner, RegistryManager stage, int
             if (oldItem instanceof BlockItem)
             {
                 @SuppressWarnings("unchecked")
-                BiMap<Block, Item> blockToItem = owner.getSlaveMap(BLOCK_TO_ITEM, BiMap.class);
+                Map<Block, Item> blockToItem = owner.getSlaveMap(BLOCK_TO_ITEM, Map.class);
```

Comment:
This alone isn't really sufficient: Mojang changed the code due to `WallOrFloorItem`, and this should really be symmetric with the following `((BlockItem)item).addToBlockToItemMap(blockToItem, item);` call.

Reply:
I'll take a look.

Reply:
Hmmm, the default implementation of `addToBlockItemMap` just does `blockToItemMap.put(this.getBlock(), itemIn);`, but I see what you mean.

Forge will need to add a corresponding `removeFromBlockItemMap` method to go along with that one.